arm/Microsoft.SignalRService/webPubSub/.test/deploy.full.test.bicep

```diff
@@ -0,0 +1,22 @@
+var location = 'westeurope'
```

Comment:
Please remove the folder for the time being. Implementing this change should be done in a consistent way across all modules